tests/test_diff_rrules.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from dateutil.rrule import MONTHLY, rrule
+
+from deepdiff import DeepDiff
+
+
+class TestDeltaIterables:
+    def test_diff_rrules(self):
```

Comment:
We can probably move this test to `test_diff_text.py`. It is not doing a delta test as its class name says it does, either.

Reply:
Yes, this was still on my earlier assumption that it had to do with Deltas. Moved and renamed.